src/components/nodes/Number.vue

```diff
@@ -13,6 +13,7 @@ export default {
     inputs: [],
     outputs: [
       {
+        id: 'output',
```

Comment:
Should this be 'number' or is 'output' appropriate for all value nodes?

Reply:
"value" would also be a possiblity.

Reply:
This applies to the connector label as well.